package.json

```diff
@@ -74,7 +74,7 @@
     "precommit": "npm run lint",
     "snapshot": "fedx-scripts jest --updateSnapshot",
     "start": "fedx-scripts webpack-dev-server --progress",
-    "test": "fedx-scripts jest --coverage --passWithNoTests"
+    "test": "fedx-scripts jest --coverage --passWithNoTests --maxWorkers=50%"
```

Comment:
What does this do, and does everyone need it?

Reply:
yeah in my prior experience this will limit the no. of cores used by jest. often jest will used all cores on a machine and test runs actually run slower with more cores, a known issue. jest's own doc recommends tuning this setting. also in CI cpus may be different count so this will auto tune jest worker count based on cpu cores. locally I have seen a couple seconds or so improvement, let's see what we find in CI